ecf/cycle_end.ecf

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -l
+
+model=%SUITE%
+%include <head.h>
+%include <envir-p1.h>
+set -x
+cyc=%CYC%
+ecflow_client --requeue=force /wafs/primary/${cyc}/wafs
```

Comment:
I don't think this will work.

Reply:
It works, however different from NCO's version. I am running ecflow parallel run and it has been worked on dogwood since 15th.